ui/pages/Project/components/RnaSeqOutliersTable.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Segment, Header } from 'semantic-ui-react'
+
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+import { GeneSearchLink } from 'shared/components/buttons/SearchResultsLink'
+import DataTable from 'shared/components/table/DataTable'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import { COVERAGE_TYPE, JUNCTION_TYPE } from 'shared/components/panel/family/constants'
+import { ButtonLink } from 'shared/components/StyledComponents'
+
+const ScrollableSegment = styled(Segment)`
+  overflow: auto;
+  max-height: 400px;
+`
+
+const RNA_SEQ_SPLICE_NUM_FIELDS = ['zScore', 'pValue', 'deltaPsi']
+const RNA_SEQ_SPLICE_LOC_FIELDS = ['chrom', 'start', 'end', 'strand']
+const RNA_SEQ_SPLICE_DETAIL_FIELDS = ['type', 'readCount', 'rareDiseaseSamplesWithJunction', 'rareDiseaseSamplesTotal']
+
+const RNA_SEQ_SPLICE_COLUMNS = [
+  {
+    name: 'geneId',
+    content: 'Gene Symbol',
+    format: row => (
+      <GeneSearchLink
+        buttonText={row.geneSymbol}
+        location={row.geneId}
+        familyGuid={row.familyGuid}
+        floated="right"
+      />
+    ),
+  },
+  {
+    content: 'Junction Search',
```

Comment:
There should be one Junction column, and it should be the first column. It's content should be a summary of the junction location (`${chrom}:${start}-${end} (${strand})`) that, when selected, opens IGV for that junction, and next to that is a search Icon that is the gene search link